README.md

```diff
@@ -148,17 +148,21 @@ Please use following command to run peaq-network-node parachian in the docker co
 #### PEAQ-Dev env
 
  ```bash
+ # First we are getting the specs files
+ wget https://github.com/peaqnetwork/async-agung-chain-specs/releases/download/async-agung-chain-specs-01-2025/async-agung-parachain-specs.json
+ wget https://github.com/peaqnetwork/async-agung-chain-specs/releases/download/async-agung-chain-specs-01-2025/async-agung-relaychain-specs.json
+
 docker run -v peaq-dev-storage/chain-data -p 9944:9944 peaq/parachain:peaq-dev-v0.0.103 \
 --parachain-id 2000 \
---chain ./node/src/chain-specs/peaq-dev-raw.json \
+--chain ./async-agung-parachain-specs.json \
```

Comment:
please also add the volume attachment; otherwise, the docker container cannot get your local file